test/testdata/deployednettemplates/networks/bootstrapped/okSpec.json

```diff
@@ -0,0 +1,8 @@
+{
+  "NumRounds":65000,
```

Comment:
don't forget that after making the Json tag names changes, you'll need to change the key names here to match.